.github/workflows/push.yml

```diff
@@ -2,16 +2,16 @@ on: push
 name: Continuous Integration
 jobs:
   test:
-    name: docker://python:3.6-stretch
+    name: docker://python:3.9-slim
```

Comment:
Upgraded Python version to 3.9, the version we use elsewhere. The `stretch` suffix as a Debian distribution that that is not published anymore with the Docker Python image. I think `buster` would be the one we'd use instead, but I selected the `slim` version instead as we need really minimal dependencies and a light Docker image.

Reply:
A bit confused... This is the name of the test or the image? I see it here
![image](https://user-images.githubusercontent.com/8228248/148272945-b6e49455-5e07-4d9d-8125-2e70712a49ad.png)

Reply:
We also should switch this to be required since GitHub isn't happy with the name change
![image](https://user-images.githubusercontent.com/8228248/148273192-d0688635-c4e3-4576-a6ca-08f569284687.png)

Reply:
Okay I googled a bit about the issue and I understand what you meant: github still expect the old action that we renamed to pass but obviously, it won't find it. I will try to disable the checks temporarily, for the time we merge this PR. We know that new action is green by manually checking ✔️